blocks/extensions/__init__.py

```diff
@@ -549,7 +566,10 @@ def do(self, which_callback, *args):
         for action in ['train', 'read_data']:
             self.previous[level][action] = self.current[level][action]
             self.current[level][action] = profile['training', 'epoch', action]
-            current_row['time_{}_this_{}'.format(action, level)] = \
-                self.current[level][action] - self.previous[level][action]
-            current_row['time_{}_total'.format(action)] = \
+            this_time = self.prefix + 'time_{}_this_{}'
+            current_row[this_time.format(action, level)] = (
+                (self.current[level][action] - self.previous[level][action]) /
+                self.divisor[level])
+            total_time = self.prefix + 'time_{}_total'
```

Comment:
It seems like we can make weird things happen here if specifying weird things like `every_n_batches=5, after_batch=True`.